docs/ansible/roles/debops.pki/acme-integration.rst

```diff
@@ -199,6 +200,10 @@ Names`_), which does not include the ``example.com`` apex domain.
         acme_domains: [ 'mon.example.com' ]
         # acme_ca: 'le-staging'
```

Comment:
You could also update this example, since the `le-staging` version is not usable anymore. Replace this with `le-staging-v2`.

Reply:
Done.